targets/why2025/badge_bsp_device.c

```diff
@@ -2,24 +2,34 @@
 // SPDX-FileCopyrightText: 2024 Nicolai Electronics
 // SPDX-License-Identifier: MIT
 
+#include "bsp/device.h"
+#include "bsp/display.h"
+#include "bsp/i2c.h"
+#include "esp_err.h"
+
 #include <stdbool.h>
 #include <stdint.h>
+
 #include <string.h>
-#include "bsp/device.h"
-#include "esp_err.h"
 
-static const char device_name[] = "WHY2025 badge";
-static const char device_manufacturer[] = "Badge.Team";
+static char const device_name[]         = "WHY2025 badge";
+static char const device_manufacturer[] = "Badge.Team";
+
+esp_err_t bsp_device_initialize(void) {
+    ESP_ERROR_CHECK(bsp_display_initialize());
```

Comment:
I understand that errors in BSP initialisation are unrecoverable. It would be nice to either change the function to not return anything and clarify in the documentation that if this fails the program will be terminated or use `ESP_RETURN_ON_ERROR`

Reply:
Fixed.